flask_resty/sorting.py

```diff
@@ -49,7 +50,7 @@ def __call__(self, query, view):
         return self.sort_query(query, view, self._field_specs)
 
 
-class Sorting(SortingBase):
+class Sorting(SortingBase, SpecDeclaration):
```

Comment:
Possibly `SortingBase` should extend `SpecDeclaration`? And just raise `NotImplementedError()` (or do nothing).